back/app/services/notifications/create_notification.rb

```
@@ -0,0 +1,37 @@
+module Notifications
+  # 個人向け通知（レビュー投稿・フォロー）を1件生成するサービス。
+  # フェーズ2/3で ActionCable によるブロードキャストや Web Push をこのサービスに追加する。
+  class CreateNotification
+    def self.call(recipient:, action:, actor: nil, notifiable: nil)
+      new(recipient:, action:, actor:, notifiable:).call
```

RuboCop (1.86.2)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)